samples/sample_common/src/vaapi_utils_drm.cpp

```diff
@@ -82,13 +82,35 @@ int open_intel_adapter(int type)
     return -1;
 }
 
-DRMLibVA::DRMLibVA(int type)
+int open_intel_adapter(int type, const char * devicePath)
+{
+    if(strcmp(devicePath, "") == 0)
```

Comment:
> why devicePath default is "" instead of nullptr? Can you, please, switch to nullptr?

Answering myself. If we use std::string to parse arguments, then c_str() don't give out nullptr as far as I understand.

However, why you don't pass std::string right here? I would suggest exactly that and convert to c_str() at the very last moment on the open() call.

Reply:
Done